tuxemon/resources/db/locale/eo.json

```diff
@@ -0,0 +1,179 @@
+{
+    "combat_run": "${{user}} forkuris!",
+    "combat_player_run": "Vi forkuris!",
+    "combat_used_x": "${{name}} estis uzata de ${{user}}!",
+    "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
+    "combat_call_tuxemon": "Iru, ${{name}}!",
+    "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
+    "combat_state_poison_damage": "${{name}} estis venenigita!",
+    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_victory": "Vi gajnis!",
+    "combat_defeat": "Vi estis venkita!",
+    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing_fail": "${{name}} liberiĝis!",
+    "combat_replacement": "Elektu anstataŭanton!",
+    "combat_replacement_is_fainted": "La monstro svenis!",
+    "combat_wild_appeared": "Sovaĝa ${{name}} aperis!",
+    "combat_monster_choice": "Kion ${{name}} faris?",
+    "combat_fainted": "${{name}} svenas!",
+    "combat_isactive": "${{name}} estas jam en ludado.",
+
+    "empty_slot": "Malplena ujo",
+    "exit": "Eliri",
+    "item_confirm_use": "Uzi",
+    "item_confirm_cancel": "Nuligi",
+    "item_failure": "Ĝi fiaskis!",
+    "item_success": "Ĝi funkciis!",
+    "attempting_capture": "Kaptado...",
+    "gotcha": "Bone!",
+
+    "item_apple_name": "Pomo",
+    "item_apple_descr": "Ĝi sanigas monstron de 10 SP.",
+    "item_berry_name": "Bero",
+    "item_berry_descr": "Ĝi sanigas monstron de 10 SP.",
+    "item_cherry_name": "Ĉerizo",
+    "item_cherry_descr": "Ĝi sanigas monstron de 10 SP.",
+    "item_orange_name": "Oranĝo",
+    "item_orange_descr": "Ĝi sanigas monstron de 20 SP.",
+    "item_potion_name": "Pocio",
+    "item_potion_descr": "Ĝi sanigas monstron de 50 SP.",
+    "item_super_potion_name": "Ega pocio",
+    "item_super_potion_descr": "Ĝi sanigas monstron de 150 SP.",
+    "item_imperial_potion_name": "Imperia pocio",
+    "item_imperial_potion_descr": "Ĝi sanigas monstron de 350 SP.",
+    "item_capture_device_name": "Kaptilo",
+    "item_capture_device_descr": "Ilo por kapti monstron.",
+    "item_cannot_use_here": "${{name}} uzeblas ĉi tie!",
```

Comment:
It looks like this string should be negated. At current it has the opposite meaning of the English string.
"${{name}} ne uzeblas ĉi tie!" seems right to me.